Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAuditdPlugin.py

```diff
@@ -266,7 +271,7 @@ def Set(WorkspaceId, Ensure):
     rules_diff = ""
     rules_diff_path = ""
     # Only do loaded rules check if auoms version < 2.0
-    if CompareVersion(auoms_version, MIN_AUOMS_VER_FOR_V3_CONFIG) < 0:
+    if CompareVersion(auoms_version, '2.0') < 0:
```

Comment:
Seems like you might want to revert this change and use: MIN_AUOMS_VER_FOR_V3_CONFIG

Reply:
I made this change on purpose to make the logic more explicit (See the code comment one line above).